### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by PookieBuns** [joshmedeski/sesh#117] [Q&A] [answered]
What is the best way to help out / contribute to this repository? I see a handful of enhancement requests, but at the same time you mentioned that you are in the process of rewriting sesh. Is there some place we can know what is open to public contribution and what is currently being worked on by you so we don't end up working on the same feature? A contributing.md document would be great!

**Comment by joshmedeski** [comment, marked as answer]
Hi @PookieBuns thanks for showing interesting in contributing! I'm continuing the rewrite work, so when I'm done with that I'll be ready to take on contributors again.

I typically [live stream on Thursdays at 7pm CST](https://www.youtube.com/channel/UCZ1y-iH9Hsp83vh6graA7Zw) so you're welcome to join the live stream and share your thoughts and ideas while I work on it live.

v2 of sesh will include the ability to fully test all aspects of the project, and I am making the connection much easier to understand and more predictable.

I'm going to open up contributing again when I finish the rewrite (hopefully in the next month or so). Typically I ask people to comment on the issue they are interested in working on and I can assign it to them and work together to work out requirements and implementation details.

When I open up contributing again I'll be sure to create a CONTRIBUTING document at that time.

p.s. you can always [join my Discord server](https://discord.gg/RQnZH7fFse) in the mean time and talk real-time about sesh and other related tings there!
